media-session/audio.js

```diff
@@ -85,6 +87,20 @@ try {
   log('Warning! The "stop" media session action is not supported.');
 }
 
+/* Seek To (supported since Chrome 78) */
+
+try {
+  navigator.mediaSession.setActionHandler('seekto', function(event) {
+    log('> User clicked "Seek To" icon.');
+    if (event.fastSeek) {
+      return;
+    }
```

Comment:
Why bailing if `fastSeek` is set?

Reply:
I thought it would be called again. What would you do instead?

Reply:
I don't really have a recommendation. I think `fastSeek` is when you scrub the timeline instead of when you tab on it. I am not sure it will end with a non-fastSeek call at the end. I don't think Chrome will ever send something with `fastSeek = true` so I can't give more insight here.

Maybe this could be linked to the fast seek call in the HTMLMediaElement?

Reply:
Shall I simply omit then from this sample until we figure out what to do with it?

Reply:
I think we should do that:
```
if (event.fastSeek && ('fastSeek' in audio)) {
  audio.fastSeek(event.seekTime);
}
```

This way, we ignore it on Chrome but not on browsers that implement `fastSeek()`. I want to be mindful of them.

Reply:
Thanks!